Sources/AppcuesKit/Presentation/Debugger/FloatingView/DebugView.swift

```diff
@@ -153,17 +154,21 @@ internal class DebugView: UIView {
             dismissView.trailingAnchor.constraint(equalTo: trailingAnchor),
             dismissView.bottomAnchor.constraint(equalTo: bottomAnchor),
 
-            toastWrapperView.leadingAnchor.constraint(equalTo: leadingAnchor),
-            toastWrapperView.trailingAnchor.constraint(equalTo: trailingAnchor),
-            toastWrapperView.bottomAnchor.constraint(equalTo: bottomAnchor)
+            toastView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 25),
+            toastView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -25),
```

Comment:
Maybe use the `readableContentGuide` here so we don't get super wide toasts on iPads?

Reply:
yes, thank you - had a pending comment about tablet and landscape that i forgot to submit, doh. but yeah exactly, will do.

Reply:
OK I think this looks reasonable now:

https://user-images.githubusercontent.com/19266448/224070636-699ef9f7-e44d-4515-a4d6-43f4298e8148.mov